src/OpenLoco/Windows/News/News.cpp

```diff
@@ -40,6 +40,7 @@ namespace OpenLoco::Ui::Windows::NewsWindow
             {
                 case Common::widx::close_button:
                 {
+                    // identical to MessageManager::clearActiveMessage()
```

Comment:
should we call `MessageManager::clearActiveMessage()` here the?

Reply:
?

Reply:
yeah probably